### DISCUSSION TITLE
classes will show inline html, but not in css and does not render in browser

### DISCUSSION THREAD

**Discussion by MBSN91** [tailwindlabs/tailwindcss#9522] [Help] [answered]
A strange bug is occurring in my project. Using the native classes from Tailwind I add f.e. my-1.
that works as intended, but when I try anything other than my-1, it won't add itself into the preview or live website. It will show up inline HTML but not in the styling panel and I cannot see it visually in the preview browser.

I have this error with z-index, bg-xxxx and some other classes. It will do one, but all the others is ignored somehow.

I've it reproduced on "hidden after question was solved"

Code in question which does not render:
bg color: hidden after question was solved
z-index: hidden after question was solved
margin top-bottom: hidden after question was solved

**Comment by wongjn** [comment, marked as answer]
It seems like your `content` globs in your `tailwind.config.js` does not fully cover all of your source code files that use Tailwind classes. In this case, it is not scanning for `.jsx` files. I would suggest adding it to your `content` globs:
```diff
 module.exports = {
-    content: ["./src/**/*.{html,js,astro,css}"],
+    content: ["./src/**/*.{html,js,jsx,astro,css}"],
     theme: { 
```

**Reply by MBSN91** [reply]
Thanks a lot! That solved it.
